spaces/aliases.json

```
@@ -36,5 +36,6 @@
 	"digital-reserve-currency": "drctoken.eth",
 	"web3-api": "polywrap.eth",
   "mstable": "mstablegovernance.eth",
-  "adex": "adex.eth"
+  "adex": "adex.eth",
+  "nftorbit": "nftorbit.eth"
```

When creating a space you do not have to edit the `aliases.json` file.
```suggestion
  "adex": "adex.eth"
```